python_modules/dagster/dagster/_core/definitions/graph_definition.py

```diff
@@ -675,6 +677,7 @@ def to_job(
                 will produce. Generally should not be set manually.
             input_values (Optional[Mapping[str, Any]]):
                 A dictionary that maps python objects to the top-level inputs of a job.
+            owners (Optional[Sequence[str]]): A sequence of strings identifying the owners of the job.
```

Comment:
probably worth adding a bit more description here (can just copy/paste from the AssetSpec desc)

```Each
            string can be a user's email address, or a team name prefixed with `team:`,
            e.g. `team:finops`.
```